.spelling

```diff
@@ -349,6 +351,8 @@ mixer-spof-myth-1
 mixer-spof-myth-2
 mongodb
 mtls_excluded_services
+multicloud
+multicluster
```

Comment:
The 0.2 annoucement blog post uses multi-cluster, could you change the announcement to use multicluster please?

Reply:
will dio.